backend/de.metas.contracts/src/main/sql/postgresql/system/50-de.metas.contracts/5722770_sys_process_contract_price_currency_readonly.sql

```diff
@@ -0,0 +1,6 @@
+-- Process: ModCntr_Specific_Price_Update(de.metas.contracts.modular.process.ModCntr_Specific_Price_Update)
+-- ParameterName: C_Currency_ID
+-- 2024-04-25T09:29:21.922Z
+UPDATE AD_Process_Para SET ReadOnlyLogic='1=1',Updated=TO_TIMESTAMP('2024-04-25 11:29:21.922','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Process_Para_ID=542802
```

Comment:
how the C_Currency_ID will be set if it's reasonly?

Reply:
On contract create, when the initial prices are set. For example here `de.metas.contracts.modular.ModularContractPriceService#createModularContractSpecificPricesFor`. The contract is called on specific price line on contract and updated there + logs